dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/stream/TripleClientStream.java

```diff
@@ -449,23 +449,25 @@ public void onHeader(Http2Headers headers, boolean endStream) {
 
         @Override
         public void onData(ByteBuf data, boolean endStream) {
-            executor.execute(() -> {
-                if (transportError != null) {
-                    transportError.appendDescription(
-                        "Data:" + data.toString(StandardCharsets.UTF_8));
-                    ReferenceCountUtil.release(data);
-                    if (transportError.description.length() > 512 || endStream) {
-                        handleH2TransportError(transportError);
-                    }
-                    return;
-                }
-                if (!headerReceived) {
-                    handleH2TransportError(TriRpcStatus.INTERNAL.withDescription(
-                        "headers not received before payload"));
-                    return;
+            executor.execute(() -> doOnData(data, endStream), () -> ReferenceCountUtil.release(data));
```

Comment:
Still using Executor

Reply:
maybe，go ahead

Reply:
Resubmitted, please review.

Reply:
Is it possible to reuse jdk's closeable?

Reply:
![image](https://github.com/apache/dubbo/assets/18413695/735a29f4-15db-4298-9cdd-d2d855766c9d)
It would throw a IOException, and SerializingExecutor need handle it, but SerializingExecutor shouldn't do the job

Reply:
Ok.fine